py_swf/clients/workflow.py

```diff
@@ -27,24 +30,32 @@ def start_workflow(self, input, id):
         :type input: string
         :param id: Freeform string that represents a unique identifier for the workflow.
         :type id: string
+        :param workflow_name: The name of the workflow type. The combination of workflow type name and version must be
+            unique with in a domain.
+        :type workflow_name: string
+        :param version: The version of the workflow type.
+        :type version: string
         :returns: An AWS generated uuid that represents a unique identifier for the run of this workflow.
         :rtype: string
         """
-        return self.boto_client.start_workflow_execution(
-            domain=self.workflow_client_config.domain,
-            childPolicy='TERMINATE',
-            workflowId=id,
-            input=input,
-            workflowType={
-                'name': self.workflow_client_config.workflow_name,
-                'version': self.workflow_client_config.workflow_version,
-            },
-            taskList={
-                'name': self.workflow_client_config.task_list,
-            },
-            executionStartToCloseTimeout=str(self.workflow_client_config.execution_start_to_close_timeout),
-            taskStartToCloseTimeout=str(self.workflow_client_config.task_start_to_close_timeout),
-        )['runId']
+        start_workflow_result = StartWorkflowResult(
```

Comment:
If this function is only returning the run_id, this namedtuple is not necessary.